README.md

```diff
@@ -86,7 +86,7 @@ Shakedown is licensed under the Apache License, Version 2.0.  For additional inf
 
 ## Reporting issues
 
-Please report issues and submit feature requests for Shakedown by [creating an issue in the DC/OS JIRA with the "Shakedown" component](https://dcosjira.atlassian.net/secure/CreateIssueDetails!init.jspa?pid=10001&issuetype=10003&components=10700).
+Please report issues and submit feature requests for Shakedown by [creating an issue in the DC/OS JIRA with the "Shakedown" component](https://jira.mesosphere.com/secure/CreateIssueDetails!init.jspa?pid=14105&components=19807&issuetype=3).
```

Comment:
Isn't that self-explainatory when you're presented with a "`You are not logged in, and do not have the permissions required to create an issue in this project as a guest.
To create an issue first log in`" screen?  :)

I'll make a quick update.